spec/services/import_polygons_spec.rb

```diff
@@ -82,191 +93,178 @@
+      let(:unchanged_buffers_response) { { "geometries" => [{ "rings" => [[["This is a polygon"]]] }] } }
+      let(:location_name) { "bath" }
+      let(:original_polygons) do
         {
-          "1" => [buffer_polygon],
-          "5" => [buffer_polygon],
-          "10" => [buffer_polygon],
-          "15" => [buffer_polygon],
-          "20" => [buffer_polygon],
-          "25" => [buffer_polygon],
-          "30" => [buffer_polygon],
-          "35" => [buffer_polygon],
-          "40" => [buffer_polygon],
-          "45" => [buffer_polygon],
-          "50" => [buffer_polygon],
-          "55" => [buffer_polygon],
-          "60" => [buffer_polygon],
-          "70" => [buffer_polygon],
-          "80" => [buffer_polygon],
-          "90" => [buffer_polygon],
-          "100" => [buffer_polygon],
-          "200" => [buffer_polygon],
+          "1" => [["This is a polygon"]],
+          "5" => [["This is a polygon"]],
+          "10" => [["This is a polygon"]],
+          "15" => [["This is a polygon"]],
+          "20" => [["This is a polygon"]],
+          "25" => [["This is a polygon"]],
+          "30" => [["This is a polygon"]],
+          "35" => [["This is a polygon"]],
+          "40" => [["This is a polygon"]],
+          "45" => [["This is a polygon"]],
+          "50" => [["This is a polygon"]],
+          "55" => [["This is a polygon"]],
+          "60" => [["This is a polygon"]],
+          "70" => [["This is a polygon"]],
+          "80" => [["This is a polygon"]],
+          "90" => [["This is a polygon"]],
+          "100" => [["This is a polygon"]],
+          "200" => [["This is a polygon"]],
         }
       end
 
       before do
         create(:location_polygon,
                name: location_name,
                location_type: api_location_type,
-               polygons: polygons,
-               buffers: original_buffers)
+               buffers: original_polygons)
       end
 
-      context "when the buffer radius options are different from the last time the task was run" do
-        let(:new_buffers) do
+      context "when the first buffer polygon retreived is the same as the last time the task was run" do
+        before do
+          allow(HTTParty).to receive(:get).with(
+            "https://tasks.arcgisonline.com/arcgis/rest/services/Geometry/GeometryServer/buffer?bufferSR=3857&"\
+            "distances=#{convert_miles_to_metres(ImportPolygons::BUFFER_RADII.first)}&f=json&geodesic=false&geometries=%7B%22geometryType%22%3D%3E%22esriGeometryPolygon%"\
+            "22%2C+%22geometries%22%3D%3E%5B%7B%22rings%22%3D%3E%5B%5B%5B51.406361958644%2C+-2.3780576677997%5D%"\
+            "2C+%5B51.4063596372237%2C+-2.3787764623145%5D%5D%5D%7D%5D%7D&inSR=4326&outSR=4326&unionResults=true&"\
+            "unit=",
+          ).and_return(unchanged_buffers_response)
+        end
+
+        it "skips the API call that returns all of the buffered polygons and returns the current buffers" do
```

Comment:
Let's use plural `API calls that return` to make it clear to future refactorers that we are saving multiple API calls with this check, so it's actually worth doing.